controllers/cloud.redhat.com/providers/featureflags/localfeatureflags.go

```diff
@@ -174,7 +174,7 @@ func makeLocalFeatureFlags(o obj.ClowdObject, objMap providers.ObjectMap, usePVC
 
 	labels := o.GetLabels()
 	labels["env-app"] = nn.Name
-
+	labels["service"] = "featureflags"
```

Comment:
Would be great to also have the app's name as `app=appName`, similarly to the `database`

Reply:
featureflags is provided for the clowdenv deployment (one FF server for the whole NS), doesn't know about specific clowdapps